docs/development/contributing_guide.md

```diff
@@ -254,8 +254,14 @@ configuration:
 ```sh
 $ docker run --rm -it -v /path/where/you/have/cloned/the/repository\:/src:ro -v /path/to/where/you/want/logs\:/logs matrixdotorg/sytest-synapse:buster
 ```
+(Note that the paths must be full paths! You could also write `$(realpath relative/path)` if needed.)
 
-This configuration should generally cover  your needs. For more details about other configurations, see [documentation in the SyTest repo](https://github.com/matrix-org/sytest/blob/develop/docker/README.md).
+This configuration should generally cover your needs.
+
+- To run with Postgres, supply the `-e POSTGRES=1 -e MULTI_POSTGRES=1` environment flags.
```

Comment:
`MULTI_POSTGRES` seems undocumented on the sytest side, but seems to run separate databases for main and state. I feel like that's a somewhat weird thing to normally test against and is probably not necessarily here?

Reply:
It seems more correct than running them in unified mode, since that means that queries can run against both (which of course won't work in multi-Postgres mode).
However this was just what I was told to run.

Reply:
> It seems more correct than running them in unified mode, since that means that queries can run against both (which of course won't work in multi-Postgres mode).

I'm not sure why it seems more correct? Can you elaborate a bit about what you mean!

Reply:
Sure, I can elaborate a bit on that last clause about what I mean!

If you have two separate Postgres databases, then fewer potential queries will work in that context than if you only had one. Put another way, a test result from a test with one unified database offers no implication that the code will work with split databases.
As a general idea, I think it's good to test things with the most restrictive configuration possible — in that sense, it'd be somewhat weird to test against the unified database (though of course there are reasons for that too, and in any case it's good to cover all the corners eventually, but maybe that explains why as a first approximation it seems less correct to test in a situation where the databases aren't isolated).

Reply:
Can we document this on the sytest side so that when someone tries to figure out what they do by following the link it says something there?